cli/src/main/kotlin/com/wire/kalium/cli/ConversationsApplication.kt

```diff
@@ -40,9 +45,37 @@ class ConversationsApplication : CliktCommand() {
                     println("ID:${it.id}, Name: ${it.name}")
                 }
             }
+
+            uploadTestAsset(networkModule)
         }
     }
 
+    private suspend fun uploadTestAsset(networkModule: AuthenticatedNetworkContainer) {
+        val imageBytes: ByteArray? = getResource("moon1.jpg")
+        val uploadResult = networkModule.assetApi.uploadAsset(
+            AssetMetadata("image/jpeg", true, AssetRetentionType.ETERNAL, calcMd5(imageBytes)),
+            imageBytes ?: ByteArray(16)
+        )
+        println("The upload result is -> $uploadResult")
+    }
+
+    @Throws(IOException::class)
+    fun getResource(name: String?): ByteArray? {
+        val classLoader = this::class.java.classLoader
+        classLoader.getResourceAsStream(name).use { resourceAsStream -> return toByteArray(resourceAsStream) }
+    }
+
+    @Throws(IOException::class)
+    fun toByteArray(input: InputStream?): ByteArray? = input?.let { inputStream ->
```

Comment:
if this code is from legacy then it was interpolated from java and in our usecase no need for it to be nullable

Reply:
Correct. I removed the nullable type